package.json

```diff
@@ -73,9 +73,12 @@
     "yamljs": "^0.3.0"
   },
   "devDependencies": {
+    "@internal/eslint-plugin": "link:.eslint",
     "all-contributors-cli": "^6.7.0",
     "auto-changelog": "^2.4.0",
-    "eslint": "^8.28.0",
+    "eslint": "^8.29.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "26.9.0",
```

Comment:
yarn moans about typescript being an unmet peer dependency.
```
warning "eslint-plugin-jest > @typescript-eslint/utils > @typescript-eslint/typescript-estree > tsutils@3.21.0" has unmet peer dependency "typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta".
```
i wonder why typescript is needed at all when we dont even have any typescript nor typescript-eslint ?

Reply:
fixed in https://github.com/fomantic/Fomantic-UI/pull/2589/commits/b5a02002458e6626d232d15a3c2a465369b789e6, `eslint-plugin-jest` needs several peer dependencies, but because of Node 12 support we cannot add Airbnb and Unicorn linter packages directly anyway, so I removed `eslint-plugin-jest` from `package.json` too and fixed the peer deps in the CI install workflow step only (the tests that need `eslint-plugin-jest` can eventually be dropped in the future, as they do not test much)